tritium-lib/src/main/java/com/palantir/tritium/proxy/Instrumentation.java

```diff
@@ -75,25 +84,6 @@ private Instrumentation() {
         return wrap(interfaceClass, delegate, InstrumentationFilters.INSTRUMENT_ALL, handlers);
     }
 
-    /**
-     * Return an instrumented proxy of the specified service interface and delegate that records aggregated invocation
-     * metrics and performance trace logging.
-     *
-     * @param serviceInterface service interface
-     * @param delegate delegate to instrument
-     * @param metricRegistry metric registry
-     * @return instrumented proxy implementing specified service interface
-     * @deprecated use {@link com.palantir.tritium.Tritium#instrument(Class, Object, MetricRegistry)}
-     */
-    @Deprecated
-    public static <T, U extends T> T instrument(Class<T> serviceInterface, U delegate, MetricRegistry metricRegistry) {
```

Comment:
release note: breaking change as folks will have to explicitly move to an appropriate TaggedMetricRegistry factory method